src/map/lua/lua_baseentity.cpp

```diff
@@ -10337,7 +10337,7 @@ inline int32 CLuaBaseEntity::addStatusEffect(lua_State *L)
 inline int32 CLuaBaseEntity::addStatusEffectEx(lua_State *L)
 {
     TPZ_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
-    TPZ_DEBUG_BREAK_IF(m_PBaseEntity->objtype == TYPE_NPC);
+    //TPZ_DEBUG_BREAK_IF(m_PBaseEntity->objtype == TYPE_NPC);
```

Comment:
at nmp point will an NPC ever have a status effect. if its gaining effects, its not an npc, it is a mob (pets are also mobs)

Reply:
Yeah this was a mistake i made when i was doing some stupid things, ive changed it back now.